### PR TITLE
terraform-ls: Update to version 0.32.7 and fix manifest

### LINKED ISSUE #127: terraform-ls: doesn't update correctly
Error from the excavator:
```text
terraform-ls: 0.32.6 (scoop version is 0.30.1) autoupdate available
Autoupdating terraform-ls
Searching hash for terraform-ls_0.32.6_windows_386.zip in https://github.com/hashicorp/terraform-ls/releases/download/v0.32.6/terraform-ls_0.32.6_SHA256SUMS
The remote server returned an error: (404) Not Found.
URL https://github.com/hashicorp/terraform-ls/releases/download/v0.32.6/terraform-ls_0.32.6_SHA256SUMS is not valid
Could not find hash in https://github.com/hashicorp/terraform-ls/releases/download/v0.32.6/terraform-ls_0.32.6_SHA256SUMS
Downloading terraform-ls_0.32.6_windows_386.zip to compute hashes!
VERBOSE: Requested HTTP/1.1 GET with 0-byte payload
VERBOSE: Received HTTP/1.1 response of content type application/json of unknown size
VERBOSE: Content encoding: utf-8
The remote server returned an error: (404) Not Found.
URL https://github.com/hashicorp/terraform-ls/releases/download/v0.32.6/terraform-ls_0.32.6_windows_386.zip is not valid
ERROR Could not update terraform-ls, hash for terraform-ls_0.32.6_windows_386.zip failed!
```

Investiguation is required

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
Fix the download urls of terraform-ls that were moved outside github
## Related Tickets & Documents
Fixes #127
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### terraform-ls

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
